### DISCUSSION TITLE
How to remove the boxes in suneditor

### DISCUSSION THREAD

**Discussion by marcelosarao** [JiHong88/SunEditor#964] [Q&A] [answered]
I have a question and answer page. I'm using SunEditor, and would like to know. How to remove the boxes, and display only the texts?
![removbox](https://user-images.githubusercontent.com/25829504/159126255-ccbd487d-c508-43a0-861a-30f4502ca708.png)

**Comment by roker15** [comment, marked as answer]
SunEditor has no box style by default. Looks like your CSS is doing this. Check CSS.
